UIKitTemplate/UIKitTemplate/MathAppViewController.swift

```diff
@@ -0,0 +1,339 @@
+// MathAppViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// MathApp main screen
+final class MathAppViewController: UIViewController {
+    private let calculator = Calculator()
+    private var operands = (first: 0, second: 0)
+    private var guessNum = 0
+
+    // SafeArea
+    private var safeAreaInsents: UIEdgeInsets {
+        if #available(iOS 15.0, *) {
+            let scene = UIApplication.shared.connectedScenes.first as? UIWindowScene
+            if let insets = scene?.windows.first?.safeAreaInsets {
+                return insets
+            }
+        }
+        return .zero
+    }
+
+    private lazy var bg: UIImageView = {
```

Comment:
Лучше поменять название, возможно подойдет такое: 
```suggestion
    private lazy var imageView: UIImage(Название что на картинке)View = {
```